README.md

```diff
@@ -88,6 +88,10 @@ openapi.validation.specification-file-path=/tmp/openapi-spec/openapi.json
 # If it is within src/main/resources/folder/my-spec.json use
 openapi.validation.specification-file-path=folder/my-spec.json
 
+# Custom log level for violations
+# Default: info
+openapi.validation.violation-log-level=error
```

Comment:
should we add an example for other possible values? (warning only is missing I suppose)

Reply:
There is code completion in the right editors. For me in intellij it tells me which ones I can use :)

Reply:
but can add it nevertheless :)